lib/gui/models/drives.js

```diff
@@ -20,103 +20,94 @@
  * @module Etcher.Models.Drives
```

Comment:
Is this still relevant? Should `drives.js` also be moved to a different directory?